subxt/Cargo.toml

```diff
@@ -61,7 +61,7 @@ unstable-metadata = []
 
 # Activate this to expose the Light Client functionality.
 # Note that this feature is experimental and things may break or not work as expected.
-unstable-light-client = ["subxt-lightclient", "tokio-stream"]
+unstable-light-client = ["subxt-lightclient", "tokio-stream", "tokio"]
```

Comment:
This means that `unstable-light-client` pulls in tokio unconditionally when enabled, but what about when the `web` feature is enabled and we want to then not pull in tokio for wasm compilation?

Maybe since none of it is used, we don't hit any compile issues anyway. I can't think of a way offhand to say "when we have `native` and `unstable-light-client`, pull in tokio too, otherwise dont' (except maybe some build step which would make understanding everything complicated).

Reply:
Yep, sadly I had to bring this in and enable my hackish patch to run the unstable backend with the light-client. I think this is going to be handled by https://github.com/paritytech/subxt/pull/1455.

Ideally, I wanted to test both the legacy-backwith the light-client and the unstable-backend. I think in the end we can just use the unstable-backend for light-client testing in our CI. I've used `cfg-aliases` for our testing module, looks good at a first glance, that may simplify a bit our feature-flags conditions

Reply:
I agree that we only need to test with the unstable backend!

Once I've done some refactoring of the light client stuff (ie the follow up to #1455) then we'll be able to configure it with an unstable backend rather than requiring tokio etc right away, but I think it's ok to do this here for now anyway, as long as the tokio dep being bought in doesn't break wasm compilation for the light client :)

Reply:
This will break the light client WASM unless you make remove the default features `"time", "rt-multi-thread"` below